app/components/documents/document-view.hbs

```diff
@@ -1,12 +1,16 @@
 {{#if (and @file (await (this.canViewConfidentialPiece)))}}
   {{#if this.isPdfDocument}}
-    <object
-      data-test-document-view-file
-      type="application/pdf"
-      data={{@inlineViewLink}}
-      class="au-u-maximize-width au-u-maximize-height auk-document-view"
-    >
-    </object>
+    {{#if @isMobile}}
+      <PdfView @filePath={{@file.namedDownloadLink}} />
```

Comment:
this should be the namedDownloadLink of piece.
the "download" button in the viewer itself is now downloading with the file name (no VR number) instead of the piece name (VR number)
Will fix on ACC